src/stories/checkbox/checked.stories.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react"
+import Checkbox from "../../components/checkbox"
+import { action } from "@storybook/addon-actions"
+
+const onChange = () => action("onChange")
+const initialValue = false
+
+export default {
+  title: "Checkbox/Checked",
+  component: Checkbox,
+  args: {
+    storyName: "",
+    value: initialValue,
+  },
+  argTypes: {
+    buttonName: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+}
+
+const Template = (args) => {
+  const [value, setValue] = useState(true)
+  return (
+    <div className="mx-30 mb-40">
+      <div className="text-2xl mb-20">Example</div>
+      <div className="w-12/12 md:w-3/12">
+        <Checkbox
+          {...args}
+          checked={value}
+          onChange={(e) => {
+            setValue(e.target.checked)
```

Comment:
imo, this state should be internal to the `Checkbox`. there should be no need to keep state externally and I would certainly not encourage it in example code. can you check if this is just a port of a prior implementation or required in your opinion?

Reply:
Do you mean that useState should not be used within the stories ?